tuxemon.py

```diff
@@ -44,8 +44,11 @@
 
 if __name__ == '__main__':
     parser = ArgumentParser()
-    parser.add_argument('-l', dest='slot', metavar='1,2,3', type=int, nargs='?',
+    parser.add_argument('-d', '--data', dest='data', metavar='resources', type=str, nargs='?',
+                        default=None, help='The data directory to use')
+    parser.add_argument('-l', '--load', dest='slot', metavar='1,2,3', type=int, nargs='?',
                         default=None, help='The index of the save file to load')
+    parser.add_argument('-s', '--starting-map', dest='starting_map', metavar='bedroom.tmx', type=str, nargs='?',
```

Comment:
`metavar` is not a default, it is an alternate name for the option, used in help or listing the options.  Please change this from `bedroom.tmx`.

https://docs.python.org/3/library/argparse.html#metavar